public/list.css

```diff
@@ -0,0 +1,43 @@
+@import url('https://fonts.googleapis.com/css?family=Fascinate+Inline|Luckiest+Guy|Great+Vibes');
+
+.list-container{
+    border-radius: 5px;
+    width:100%
+}
+
+
+.widget-container svg{
+    width:30px !important;
+    height:30px !important;
```

Comment:
Why do youi use important? Never do this

Reply:
I added it to override some bootstrap styling, forgot to remove it after removing bootstrap